returnn/training.py

```diff
@@ -159,13 +180,21 @@ def __init__(
             for k in stored_epochs
             if k in self.keep_epochs
         }
-        if self.returnn_config.get("use_tensorflow", False):
+        if self.returnn_config.get("use_tensorflow", False) or self.returnn_config.get("backend") == "tensorflow":
```

Comment:
```suggestion
        if self.returnn_config.get("use_tensorflow", False) or self.returnn_config.get("backend", None) == "tensorflow":
```
All other places that call `self.returnn_config.get()` define a default value.